e2e/rbd_helper.go

```diff
@@ -30,6 +30,40 @@ func rbdOptions(pool string) string {
 	return "--pool=" + pool
 }
 
+type errGettingFsid string
+type errListingRBDNs string
+type errCreatingRBDNs string
+type errMsg string
+type getOmapVal string
```

Comment:
can we replace it with `errGetOmapVal` to match the theme of other errors?

Reply:
yes. Will change this.